apps/api/src/notifications/helpers/truncateNameToBytes.ts

```diff
@@ -0,0 +1,21 @@
+export function truncateNameToBytes(name: string, maxBytes: number) {
+  const encoder = new TextEncoder()
```

Comment:
can we add some comments about what do those Encoders do?
I guess this does something like utf8 to bytes... and whole play is not to cut in the middle of a symbol.